src/Database/Database.php

```diff
@@ -1265,14 +1263,14 @@ public function rollback(): bool
 	 * @return boolean was the delete successful?
 	 * @throws \Exception
 	 */
-	public function delete($table, array $conditions): bool
+	public function delete(string $table, array $conditions): bool
```

Comment:
Please update the documentation to reflect the type change with the optional schema prefix.